### DISCUSSION TITLE
Sort vault items by name/date

### DISCUSSION THREAD

**Discussion by danielporto** [dani-garcia/vaultwarden#3417] [Ideas]
Please add an option to sort the vault items by name (a-z, z-a or by creation/update dates) on the web interface.
The current order makes little sense, and it is easy to duplicate items just because their names are not visible next to each other.

**Comment by BlackDex** [comment]
This is a client issue, which are not maintained by Vaultwarden.
Also, this has already been reported here before, and there is already an issue open at Bitwarden.
https://github.com/dani-garcia/vaultwarden/discussions/3005
https://github.com/bitwarden/clients/issues/4263
